src/utils/container-runtime/src/runtime.rs

```diff
@@ -119,6 +119,12 @@ impl ContainerRuntime {
         Ok(())
     }
 
+    pub fn custom_cmd(&self, command: String) -> Command {
```

Comment:
I would prefer to have a `version` or `info` methods instead. This type should isolate the user from knowing specific flags and arguments.